Samples.xcodeproj/project.pbxproj

```diff
@@ -522,6 +527,9 @@
 			dstPath = "";
 			dstSubfolderSpec = 7;
 			files = (
+				9503056F2C46ED010091B32D /* ShowDeviceLocationUsingIndoorPositioningView.Model.swift in Copy Source Code Files */,
+				9503056C2C46EC8A0091B32D /* ShowServiceAreaView.swift in Copy Source Code Files */,
```

Comment:
Can you move this change into a separate PR? Thanks.

Reply:
You meant just the ShowServiceAreaView one correct?

Reply:
Right. We don't include change that has side effects in a PR even if it is trivial. Because if the PR is reverted in the future, the unrelated change will be reverted as well, which might cause problems.
Instead, create a quick fix PR is better.

Reply:
Okay I'll create quickfix pr

Reply:
Looks like it is already fixed in v.next